mix.exs

```diff
@@ -21,8 +21,7 @@ defmodule HPack.Mixfile do
 
   defp deps do
     [
-      {:earmark, "~> 0.1", only: :dev},
-      {:ex_doc, "~> 0.11", only: :dev},
+      {:ex_doc, ">= 0.0.0", only: :dev},
```

Comment:
I think it should be pinned to a version that HPack requires. And TBH I do not know which one that is, but if you specify it like this, you should make sure it works with that version. because the lock file is ignored when HPack is a dependency AFAIK. And HPack will be a dependency in almost 99% of the cases.